xds/src/main/java/io/grpc/xds/sds/internal/SdsProtocolNegotiators.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.sds.internal;
+
+import io.grpc.netty.GrpcHttp2ConnectionHandler;
+import io.grpc.netty.InternalNettyChannelBuilder;
+import io.grpc.netty.InternalNettyChannelBuilder.ProtocolNegotiatorFactory;
+import io.grpc.netty.InternalProtocolNegotiator;
+import io.grpc.netty.InternalProtocolNegotiator.ProtocolNegotiator;
+import io.grpc.netty.NettyChannelBuilder;
+import io.netty.channel.ChannelHandler;
+import io.netty.util.AsciiString;
+
+/**
+ * An internal accessor for {@link ProtocolNegotiator}s  similar to
+ * {@link io.grpc.netty.InternalProtocolNegotiators}. A gRPC {@link ProtocolNegotiator} is used
+ * to provide SSL context using certs/keys provided by SDS.
```

Comment:
this is not internal accessor. it is actual implementation of ProtocolNegotiators for SDS, can you update this doc?